.storybook/webpack.config.js

```diff
@@ -1,21 +1,21 @@
 module.exports = {
-  name: "client",
-  target: "web",
+  name: 'client',
+  target: 'web',
   resolve: {
-    extensions: [".js", "jsx"]
+    extensions: ['.ts', '.tsx', '.js', 'jsx']
```

Comment:
`jsx` is the only one missing a dot here. Is that correct? Nevertheless I would love to make it cnsistent.